modules/benchmark/R/metric.residual.plot.R

```diff
@@ -5,7 +5,7 @@
 ##' 
 ##' @author Betsy Cowdery
 
-metric.residual.plot <- function(dat, var,filename){
+metric.residual.plot <- function(dat, var,filename = NA, draw.plot = FALSE){
```

Comment:
I don't follow the logic of the draw.plot option and why drawing the plot isn't just an else on the if(filename), especially since you aren't returning anything.  Right now the DEFAULT behavior of these functions is to do nothing (neither make a pdf or a plot), which is very unintuitive.